apis/tags/v1beta1/tagbinding_types.go

```diff
@@ -24,6 +26,7 @@ var TagsTagBindingGVK = GroupVersion.WithKind("TagsTagBinding")
 // TagsTagBindingSpec defines the desired state of TagsTagBinding
 // +kcc:spec:proto=google.cloud.resourcemanager.v3.TagBinding
 type TagsTagBindingSpec struct {
+	Location *string `json:"location,omitempty"`
 	// +kcc:ref=Project
 	ParentRef *ParentRef `json:"parentRef"`
```

Comment:
It's unfortunate that this is called parentRef and not projectRef or parentProjectRef